src/compiler/emitter.ts

```diff
@@ -372,21 +398,37 @@ namespace ts {
             writer = previousWriter;
         }
 
-        function writeBundle(bundle: Bundle, output: EmitTextWriter) {
+        function writeBundle(bundle: Bundle, output: EmitTextWriter, bundleInfo?: BundleInfo) {
             isOwnFileEmit = false;
             const previousWriter = writer;
             setWriter(output);
             emitShebangIfNeeded(bundle);
             emitPrologueDirectivesIfNeeded(bundle);
+            for (const prepend of bundle.prepends) {
+                print(EmitHint.Unspecified, prepend, /*sourceFile*/ undefined);

```

Comment:
These should still be emitted after helpers (and synthetic triple slash references), as these prepends will also need the helpers and will fail at runtime if the helpers are not assigned first.

Reply:
Also just use `emit(prepend)` - `print` with no sourceFile parameter and an `Unspecified` hint is just an indirection to `emit`.